Cargo.toml

```diff
@@ -29,7 +29,7 @@ flow_scheduler = []
 lttng = ["dep:lttng-ust", "dep:lttng-ust-generate"]
 rtlsdr = ["seify/rtlsdr"]
 hackrf = ["seify/hackrfone"]
-seify = ["dep:seify"]
+seify = ["dep:seify", "futuresdr-types/seify"]
```

Comment:
Assuming if you have `seify` enabled in `futuresdr`, you'd also want the `Pmt` conversion support.